src/api/v3/feedback/search_result_feedback.py

```diff
@@ -0,0 +1,13 @@
+from django.http import HttpResponse
+
```

Comment:
I got a csrf-error when testing the api, so we might add this decorator:
```suggestion
from django.views.decorators.csrf import csrf_exempt
```